docs/kyma-environment-broker/03-10-orchestration.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Orchestration
+type: Details
+---
+
+Orchestration is an abstraction that allows you to aggregate Kyma and cluster upgrades. To create an orchestration, [send a request to the proper KEB handler](#tutorials-upgrade-kyma-runtime-using-keb).
+
+After sending the request, the orchestration is processed by KymaUpgradeManager which resolves Shoots in the Gardener cluster and filters them using input provided in the request body.
+
+When Orchestration knows its targets, it starts to process them in a separate queue as the Upgrade Operation leveraging the [upgrade steps](#details-runtime-operations) logic. 
```

Comment:
```suggestion
Then, the `KymaUpgradeManager` proceeds with the orchestration and performs the [upgrade steps](#details-runtime-operations) logic on the Runtimes that you have specified. 
```

Reply:
We can also mention here that for now, we can upgrade only one Runtime per request.